apps/web/client/src/components/ui/settings-modal/with-project.tsx

```diff
@@ -174,20 +199,7 @@ export const SettingsModalWithProjects = observer(() => {
                                                             }
                                                         >
                                                             {tab.icon}
-                                                            <Tooltip>
-                                                                <TooltipTrigger asChild>
-                                                                    <span className="truncate">
-                                                                        {capitalizeFirstLetter(
-                                                                            tab.label.toLowerCase(),
-                                                                        )}
-                                                                    </span>
-                                                                </TooltipTrigger>
-                                                                <TooltipContent>
-                                                                    {capitalizeFirstLetter(
-                                                                        tab.label.toLowerCase(),
-                                                                    )}
-                                                                </TooltipContent>
-                                                            </Tooltip>
+                                                            <TruncatedLabelWithTooltip label={capitalizeFirstLetter(tab.label.toLowerCase())} />
                                                         </Button>
```

Comment:
@Kitenite not sure if this is how you'd prefer it to be done but check it out and decide.